.github/workflows/codeql.yml

```diff
@@ -21,20 +21,30 @@ jobs:
 
     steps:
       - name: Check out repository
-        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 #v3.5.3
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 #v4.1.1
         with:
           fetch-depth: 2
 
+      - name: Download benchmark bundle
+        env:
+           GH_TOKEN: ${{ github.token }}
+        # download the latest version
+        run: |
+           gh release download -R github/codeql-action --pattern 'codeql-bundle.tar.gz'
```

Comment:
We can specify a tag to lock to a version or leave out the tag and always get the latest release.
It's not clear from the command-line docs if `gh release download` will work with the SHA hashes of the release so skipping that and just letting it use the latest release.